forge-gui/res/cardsfolder/upcoming/the_foretold_soldier.txt

```diff
@@ -0,0 +1,10 @@
+Name:The Foretold Soldier
+ManaCost:2 G G
+Types:Creature Alien Zombie Soldier
+PT:6/6
+K:Foretell:1 G
+K:CARDNAME must be blocked if able.
+S:Mode$ MinMaxBlocker | ValidCard$ Card.Self | Max$ 1 | Description$ CARDNAME can't be blocked by more than one creature.
+T:Mode$ DamageDone | ValidSource$ Card.Self | Execute$ TrigExile | TriggerDescription$ Whenever CARDNAME deals damage, exile it face down. It becomes foretold.
```

Comment:
wrong type

Reply:
Hmm, is this missing a param or is the triggertype wrong ?

Reply:
wrong triggertype